.travis.yml

```diff
@@ -5,8 +5,6 @@ sudo: false
 
 matrix:
   include:
-    - env: CABALVER=1.18 GHCVER=7.8.4
-      addons: {apt: {packages: [cabal-install-1.18,ghc-7.8.4], sources: [hvr-ghc]}}
```

Comment:
`aeson-iproute` wasn't compiling on old GHC. I think it's fine to drop this—GHC 7.8 dates back to 2014